lib/ui/common/Dogtag.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+
+class DogTagViewModel {
+  String title;
+  String number;
+  IconData icon;
+
+  DogTagViewModel({this.title, this.icon, this.number});
+}
+
+DogTagViewModel buildDogTagViewModel(
+    {String title, IconData icon, String number}) {
+  return DogTagViewModel(title: title, icon: icon, number: number);
+}
+
+class DogTagStyle {
+  final Color backgroundColor;
+  final BorderRadius borderRadius;
+  final EdgeInsets edgePadding;
+  final TextStyle titleTextStyle;
+  final int maxLines;
+  final double iconSize;
+  final double spacing;
+
+  DogTagStyle(
+      {this.backgroundColor,
+      this.borderRadius,
+      this.edgePadding,
+      this.titleTextStyle,
+      this.maxLines,
+      this.iconSize,
+      this.spacing});
+
+  factory DogTagStyle.defaultStyle() {
+    return DogTagStyle(
+        backgroundColor: Color(0x1425df0c),
+        borderRadius: BorderRadius.all(Radius.circular(20.0)),
+        edgePadding: EdgeInsets.only(top: 8.5, right: 12, left: 12, bottom: 8),
+        titleTextStyle: TextStyle(
+            fontSize: 15,
+            fontWeight: FontWeight.normal,
+            color: Color(0xff25df0c)),
+        maxLines: 1,
+        iconSize: 8,
+        spacing: 5.5);
+  }
+
+  factory DogTagStyle.positiveStyle() {
+    return DogTagStyle.defaultStyle().copyWith();
+  }
+
+  factory DogTagStyle.negativeStyle() {
```

Comment:
don't have specific use cases in a generic widget. (negative and positive style makes no sense generically)
Just have them defined were they are used or if shared in their own file.

Reply:
The styles will be refactored in a coming PR